src/input.rs

```diff
@@ -95,6 +95,10 @@ pub struct Opts {
     #[structopt(short, long, default_value = "1500")]
     pub timeout: u32,
 
+    /// The number of tries before a port is assumed to be closed.
+    #[structopt(long, default_value = "1")]
+    pub tries: u8,
```

Comment:
We also need add it to the line below so we can infer this information from the config file. :) 

 https://github.com/RustScan/RustScan/blob/5b35a09b0b449d77106668746a1a0b3a8c7d2c5e/src/input.rs#L156

Reply:
@bergabman I add this on my local version.